src/main/scala/ing/wbaa/druid/auth/basic/BasicAuthenticationExtension.scala

```diff
@@ -0,0 +1,56 @@
+package ing.wbaa.druid.auth.basic
```

Comment:
Please add the Apache 2.0 license header: https://github.com/ing-bank/scruid/blob/master/src/main/scala/ing/wbaa/druid/DruidConfig.scala#L1-L16